squash/dashboard/models.py

```diff
@@ -22,9 +22,8 @@ class Job(models.Model):
     ci_url = models.URLField(null=False, help_text='Jenkins job URL')
     status = models.SmallIntegerField(default=STATUS_OK,
                                       help_text='Job status, 0=OK, 1=Failed')
-
-    def get_jobs(self):
-        pass
+    data = JSONField(null=True, blank=True, default=None,
+                     help_text='Data produced by the job.')
```

Comment:
Clarify in the help text that this is a `validate_base` `Blob` https://validate-base.lsst.io/py-api/lsst.validate.base.BlobBase.html#lsst.validate.base.BlobBase

Maybe the name could be changed, since datasets might have a different meaning that. `validate_base_blobs`? Or just `blobs`?